ganga/GangaGUI/gui/routes.py

```diff
@@ -1,22 +1,34 @@
 import jwt
-from GangaGUI.gui import app
-from flask import request, jsonify
+import json
 from functools import wraps
+from itertools import chain
+from flask import request, jsonify
+from GangaGUI.gui import app
 from GangaGUI.gui.models import User
 
 
+# ******************** View Routes ******************** #
+
+# Dashboard route
 @app.route("/")
 def dashboard():
-    return "Hello GangaGUI"
+    return "GangaGUI"
```

Comment:
Is this a placeholder, shouldn't it be returning the main dashboard?

Reply:
Yes it is actually placeholder, I am working on the view route as of now